packages/cli/src/commands/doctor/healthchecks/reactNativeGlobal.ts

```diff
@@ -15,6 +14,11 @@ const packageManagers = {
   npm: 'npm',
 };
 
+const packages = {
```

Comment:
Is there a reason to use an object instead of array?

You are not using the entries anywhere so you can just turn this into an array and remove the need of `Object.keys` in line 35 and 37.